packages/places/src/Places.ts

```diff
@@ -28,15 +28,34 @@ interface IPlaces {
   clear: () => void;
   setAuthorizationStatus: (authStatus?: PlacesAuthStatus) => void;
 }
+ 
+const logNativeModules = () => {
```

Comment:
Let's remove the test code before merging the pull request.

Reply:
removed